src/scripts/form-validation.js

```diff
@@ -0,0 +1,107 @@
+const emailEl = document.querySelector('#email');
+const userName = document.querySelector('#Name');
+const userLastName = document.querySelector('#Last_Name');
+const message = document.querySelector('#message');
+const form = document.querySelector('#Form-form');
+
+const isEmailValid = (email) => {
+  const re = /[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/;
+  return re.test(email);
+};
```

Comment:
- [ ] Kindly work on the change the previous reviewers requested you do.
I have entered the same email you entered in the screenshot you commented on the project and the form is still submitted. It is required that the form only submit a lowercased email address and there must be no uppercase letter in the email. Kindly fix that and be sure it is working as required, then your project will be approved upon your next review
![image](https://user-images.githubusercontent.com/50941074/162083776-7a94462f-130c-43fb-83be-bb772c789275.png)